runtime/sema/authaccount.cdc

```diff
@@ -0,0 +1,297 @@
+
+pub struct AuthAccount {
+
+    /// The address of the account.
+    pub let address: Address
+
+    /// The FLOW balance of the default vault of this account.
+    pub let balance: UFix64
+
+    /// The FLOW balance of the default vault of this account that is available to be moved.
+    pub let availableBalance: UFix64
+
+    /// The current amount of storage used by the account in bytes.
+    pub let storageUsed: UInt64
+
+    /// The storage capacity of the account in bytes.
+    pub let storageCapacity: UInt64
+
+    /// The contracts deployed to the account.
+    pub let contracts: AuthAccount.Contracts
+
+    /// The keys assigned to the account.
+    pub let keys: AuthAccount.Keys
+
+    /// The inbox allows bootstrapping (sending and receiving) capabilities.
+    pub let inbox: AuthAccount.Inbox
+
+    /// All public paths of this account.
+    pub let publicPaths: [PublicPath]
+
+    /// All private paths of this account.
+    pub let privatePaths: [PrivatePath]
+
+    /// All storage paths of this account.
+    pub let storagePaths: [StoragePath]
+
+    /// **DEPRECATED**: Use `keys.add` instead.
+    ///
+    /// Adds a public key to the account.
+    ///
+    /// The public key must be encoded together with their signature algorithm, hashing algorithm and weight.
+    pub fun addPublicKey(_ publicKey: [UInt8])
+
+    /// **DEPRECATED**: Use `keys.revoke` instead.
+    ///
+    /// Revokes the key at the given index.
+    pub fun removePublicKey(_ index: Int)
```

Comment:
These functions got removed in Stable Cadence.

Reply:
Removed here and from docs: 3d57ec19028c88e752ffb8fff1ce0a4dce49b48f